smos-docs-site/pages/concepts.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: SMOS Concepts
+date: 2020-04-08
+---
+
+GTD
+: GTD is a time management method by David Allen (see [Getting This
+  Done](https://en.wikipedia.org/wiki/Getting_Things_Done) on
+  Wikipedia).
+  SMOS is supposed to be a tool to implement GTD.
+
+
+<!-- the next concepts are related to SMOS configuration, but AFAIK
+are not explained -->
+
+Workflow
+: What's the meaning in SMOS and why the need for a directory?
+
+
+Projects
+: Whats the meaning in SMOS?  Why the need for a directory in the
+  configuration file?
+
+
+Archive:
+: When task and projects are completed, they are archived (i.e. moved
```

Comment:
That's right. We archive on the file-level of granularity.
Then `smos-query` won't pick up on these files for most commands, but not `smos-query clock` for example.

Reply:
I would add a *Suggested directories layout* in the Concepts page (or in a different page?):

```
~/workflow
~/workflow/projects
~/workflow/archive
```

Is it ok for you?

Reply:
Those are the defaults, but yes they should be documented!